### DISCUSSION TITLE
Ολοκλήρωση εργασίας/απορία

### DISCUSSION THREAD

**Discussion by geopala** [courses-ionio/sw#1019] [Announcements]
Καλησπέρα, κύριε @epidrome 
Από λάθος μου ζήτησα επανεξέταση στο issue https://github.com/ioniodi/sitegr/issues/219 μετά τις αλλαγές που ζητήσατε, έχω λάβει κανονικά έγκριση από τoν βοηθό του μαθήματος. Είναι εύκολο να κάνετε approve και από μεριά σας για να γίνει εν τέλη το merge του PR?
Αφορά το https://github.com/ioniodi/sitegr/pull/222
Επίσης, όσο αφορά το 2ο αίτημα στην ιστοσελίδα, έχοντας λάβει **_good first issue_**, μπορώ να κατασκευάσω το αρχείο και να ανοίξω επόμενο PR εφόσον κλείσει του αιτήματος 1?
Ευχαριστώ προκαταβολικά

**Comment by epidrome** [comment]
https://github.com/ioniodi/sitegr/issues/220
